runtime/kusama/src/weights/frame_election_provider_support.rs

```diff
@@ -48,20 +48,20 @@ impl<T: frame_system::Config> frame_election_provider_support::WeightInfo for We
 	/// The range of component `t` is `[500, 1000]`.
 	/// The range of component `d` is `[5, 16]`.
 	fn phragmen(v: u32, _t: u32, d: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 56_000
-			.saturating_add(Weight::from_ref_time(13_944_000 as u64).saturating_mul(v as u64))
-			// Standard Error: 4_876_000
-			.saturating_add(Weight::from_ref_time(2_223_649_000 as u64).saturating_mul(d as u64))
+		Weight::from_ref_time(5_800_910_000 as u64)
+			// Standard Error: 141_466
+			.saturating_add(Weight::from_ref_time(5_793_269 as u64).saturating_mul(v as u64))
+			// Standard Error: 14_463_035
+			.saturating_add(Weight::from_ref_time(1_592_527_344 as u64).saturating_mul(d as u64))
 	}
 	/// The range of component `v` is `[1000, 2000]`.
 	/// The range of component `t` is `[500, 1000]`.
 	/// The range of component `d` is `[5, 16]`.
 	fn phragmms(v: u32, _t: u32, d: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 79_000
-			.saturating_add(Weight::from_ref_time(14_480_000 as u64).saturating_mul(v as u64))
-			// Standard Error: 6_844_000
-			.saturating_add(Weight::from_ref_time(2_525_332_000 as u64).saturating_mul(d as u64))
+		Weight::from_ref_time(4_696_655_000 as u64)
+			// Standard Error: 150_968
+			.saturating_add(Weight::from_ref_time(5_769_927 as u64).saturating_mul(v as u64))
+			// Standard Error: 15_434_495
+			.saturating_add(Weight::from_ref_time(1_820_652_967 as u64).saturating_mul(d as u64))
```

Comment:
big changes here

Reply:
The worst-case improved for both runtimes, rococo is unchanged.: 
![image](https://user-images.githubusercontent.com/10380170/191365162-9439713b-9ec6-4566-aa26-d96e7ecdb4b8.png)

Reply:
It does look like this weight was affected by the zero base weight issue that my PR fixed; looks correct to me. As @ggwpez pointed out this actually lowered the weight final weight since the negative base weight isn't rounded up to zero anymore (so we were overestimating how costly this extrinsic was when the components are not zero)